### ISSUE TITLE
Table not showing

### ISSUE THREAD

**Issue by nyokhix** [hackerain/sphinxcontrib-excel-table#6]
After following the instructions the table does not show in the Sphinx index.

when i add the witdh:1000 parameter i get the following error

```
 N:\Projects\2018\TWD-NL\TWD-NL-2018-057 TWD Dashboard And Register Upgrade\Documentatie\Documentatie\source\powerbi.rst:15: WARNING: Error in "excel-table" directive:
 unknown option: "width".
 
 .. excel-table::
    :file: /uploads/powerbi/fillinfo.xlsx
    :width: 1000
```

**Comment by nyokhix** [comment]
it does show in the HTML file but not on page

```
<p>fill in the following</p>
<div id="excel-/uploads/powerbi/fillinfo.xlsx-sheet-fill_info-icnt-1" style="margin-bottom: 24px"></div>
<script>
  var container = document.getElementById('excel-/uploads/powerbi/fillinfo.xlsx-sheet-fill_info-icnt-1');
    var hot = new Handsontable(container, {
    data: [["From Web", null, null, null], [null, null, null, null], ["Basic", "Advanced", null, null], [null, null, null, null], ["Url Parts", null, null, null], ["https://87674.resttest.afas.online/ProfitRestServices/connectors/\"name of connector\"", null, null, null], [null, null, null, null], ["Add part", null, null, null], ["URL preview", null, null, null], [null, null, null, null], [null, null, null, null], ["Command timeout in minutes (optional)", null, null, null], [null, null, null, null], ["HTTP requeset header parameters (optional", null, null, null], ["ContentType", "application/json;charset=utf-8", null, null], ["Authorization", "AfasToken PHRva2VuPjx2ZXJzaW9uPjE8L3ZlcnNpb24+PGRhdGE+MjgxQ0IwODY1RTc0NEY0REJEMDk5NDM4QjQ2MUNERDczRjlFOEZDRTQ5RDJEQjMyN0QyQUYwOEU2RkMzMjAzODwvZGF0YT48L3Rva2VuPiAg", null, null], [null, null, null, null], [null, null, null, null], [null, null, "OK", "Cancel"]],
    preventOverflow: 'horizontal',
    readOnly: true,
    width: undefined,
    colWidths: undefined,
    manualColumnResize: true,
    manualRowResize: true,
    rowHeaders: true,
    colHeaders: true,
    mergeCells: [{"row": 5, "col": 0, "rowspan": 1, "colspan": 2}]
    });
</script>
```

**Comment by hackerain** [comment]
Hi, sorry for this, this is a document error, should use **tablewidth** instead of **width**, and I have updated the document and published it to 1.0.8 version.
